Source/Analytics/Web/src/assets/main.scss

```diff
@@ -25,26 +26,19 @@ body {
 }
 
 .headerText {
+  font-family: "Lato", sans-serif;
   color: black !important; //these are marked as !important to override Google's MuiTable styling
-  font-weight: 200;
-
-  &.center {
-    text-align: center !important;
-  }
+  padding: 0.3rem !important;
```

Comment:
Do we need to have this? 😢

Reply:
Unfortunately yes 😭 It's the easiest way to override MaterialUI's own styles.
However, there are much better ways, but they require more setup. See [here](https://material-ui.com/styles/basics/).
We could add an issue for implementing one of those solutions, it could be nice to have a more uniform way of styling throughout the system.